jobs/keepalived/templates/bin/utils/openstack/port_operations

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#sourcing job properties to bring openstack credential in env variables
+# shellcheck disable=SC1091
+source /var/vcap/jobs/keepalived/bin/job_properties.sh
+jq=/var/vcap/packages/jq/bin/jq
+
+generate_body_for_port_update(){
+if [ -n "$1" ]; then
+cat << EOF
+{
+    "port": {
+        "allowed_address_pairs": [
+          {
+	  	"ip_address": "${1}"
+	  }
+        ]
+    }
+}
+EOF
+else
+cat << EOF
+{
+    "port": {
+        "allowed_address_pairs": []
+    }
+}
+EOF
+fi
+}
+
+get_port_id(){
+    local fixed_ip
+    fixed_ip="${1}"
+    local neutron_port_id
+    neutron_port_id="$(curl --retry 2 -sk -X GET \
+        -H "X-Auth-Token: ${OS_TOKEN}" \
+        -H "Accepts: application/json" \
+        "${NETWORK_PUBLIC_URL}v2.0/ports" | $jq \
+        -r ".ports[] | select(.fixed_ips[].ip_address==\"${fixed_ip}\") | .id")"
+    RET=$?
+    if [[ $RET -ne 0 ]] ; then
+        echo "$(date) [ERROR] - HTTP Error: ${RET}"
+        exit ${RET}
+    fi
+    echo "${neutron_port_id}"
```

Comment:
can we append this echo with date and some meaningful content like `neutron_port_id: ${neutron_port_id}`

Reply:
this is used for return of function. can't add anything other than port id.

Reply:
understood!